src/components/Location/location.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {shallow} from "enzyme";
+import Location from "./index";
+
+
+
+describe ("Location Component render", () => {
+   it("renders correctly", () => {
+       const wrapper = shallow(<Location name="The name of the event" />);
+       expect(wrapper).toMatchSnapshot();
+    })
+
+    it("renders correctly", () => {
```

Comment:
@AnthoniaOkafor, can we have a test that checks that the correct text renders on the page?

Reply:
Ok @meekg33k , I will do just that